embrace-gradle-plugin/src/main/java/io/embrace/android/gradle/plugin/tasks/ndk/NdkUploadTasksRegistration.kt

```diff
@@ -101,12 +101,13 @@ class NdkUploadTasksRegistration(
                 task.compressedSharedObjectFilesDirectory.asFile.get().exists() &&
                     task.architecturesToHashedSharedObjectFilesMapJson.asFile.get().exists()
             }
+            // TODO: An error thrown in registration will make the build will fail. Should we use failBuildOnUploadErrors for this too?
```

Comment:
IMO no as it's not about failing to make a HTTP request. I think we should just default to failing the build if something goes that wrong